README.md

```diff
@@ -1,45 +1,67 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Webpack-template
+# To Do list
 
-> This is a ready webpack template that can be used when you start a new project.
+> Build a To Do List JavaScript application
+![app-screenshot](./app-screenshot.png)
 
-### Prerequisites
+## About The Project
+
+> The project I have built is based on a simple app which adds tasks, marks them as completed and deletes all completed tasks. This project has been built with the help of modern CSS3, Bootstrap 5, JavaScript and webpack to bundle JavaScript. Check the live demo to try.
+
+## Built With
+
+- JavaScript, HTML, CSS
+- Webpack
+
+## Getting Started
 
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```

Comment:
- [ ] It would be nice to try to remove all unrelated notes, that doesn't give many contexts to our project and try to structure all content of documentation to suit our project what do you think? 🤔